Automattic/jetpack
# Widgets: Porting of WP.com's EU Cookie Law Banner

This is my first PR in the effort of filling the gaps between Jetpack and WP.com, to smooth the AT process out.

I'm not completely sure about the Jetpack coding standards (file, classes and variables naming conventions, documentation) as apparently different widgets have different standards, so if you have any suggestions I'll be happy to implement them ASAP.

#### Changes proposed in this Pull Request:

* Adds the EU Cookie Law Banner widget as already available on WP.com.

#### Testing instructions:

* Add the widget to a sidebar and check if the banner shows up near the bottom of the window.
* Check if, when closed with the selected method, it actually stays closed.
* Check if both light and dark themes work appropriately.
